processor_function/shared/cnab_builder.py

```diff
@@ -113,9 +116,31 @@ def deploy_aci(self):
 
         credential = AzureIdentityCredentialAdapter()
         aci_client = ContainerInstanceManagementClient(credential, self._subscription_id)
+
+        service_bus = ServiceBus()
+        service_bus.send_status_update_message(self._id, "ACI Deployment started", "Deploying ACI container: " + self._container_group_name)
+
         result = aci_client.container_groups.create_or_update(self._resource_group_name, self._container_group_name,
                                                               group)
 
         while result.done() is False:
             logging.info('-- Deploying -- ' + self._container_group_name + " to " + self._resource_group_name)
             time.sleep(1)
+
+        service_bus.send_status_update_message(self._id, "ACI container deployed", "")
+
+        logs = aci_client.container.list_logs(self._resource_group_name, self._container_group_name, self._container_group_name)
+
+        while "Error" not in logs.content and "Success" not in logs.content:
+            time.sleep(5)
+            logs = aci_client.container.list_logs(self._resource_group_name, self._container_group_name, self._container_group_name)
+            if "Error" in logs.content:
+                service_bus.send_status_update_message(self._id, "Deployment failed", logs.content)
+                print(logs.content.split("Error", 1)[1])
```

Comment:
Now that #272 has been merged, these print statements should be replaced with logging.<log level>(). This comment applies to all print statements.

Reply:
changed